.babelrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
What effect does this file have?

Reply:
Yes - the fs.go file does it currently (live for dev, pregenerated for prod), since I was unused to node/react and didn't want the dependency hell that I always find myself in when using those before.

Now I've sort of accepted that it's probably better to use standard node/react workflows, and would be very happy with changing over which it seems your PR is in the process of doing.